commonui/src/main/java/com/hyperwallet/android/ui/common/view/HorizontalDividerItemDecorator.java

```diff
@@ -31,11 +31,14 @@
 public class HorizontalDividerItemDecorator extends RecyclerView.ItemDecoration {
 
     private static final short DEFAULT_PADDING = 0;
+    private static final short TOP_DIVIDER_PADDING = 30;
```

Comment:
I'm not sure of the implications of hardcoding this value. Maybe we could try getting the value from a resource, e.g Resources.getSystem().getDimension()

Reply:
yep make sense moved the value to resource instead